src/components/ContactsList/Contacts/ContactName.jsx

```diff
@@ -1,21 +1,30 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const ContactName = ({ firstname, lastname }) => (
-  <div className="u-ellipsis">
-    <span className="contact-firstname">{firstname}</span>
-    &nbsp;
-    <span className="contact-lastname">{lastname}</span>
-  </div>
-)
+const ContactName = ({ displayName, lastname }) => {
+  const namesToDisplay = (displayName && displayName.split(' ')) || []
+  return (
+    <div className="u-ellipsis u-ml-1">
+      {namesToDisplay.map((name, key) => (
+        <span key={`display-${key}`}>
```

Comment:
Do we need the extra `span`? Would something like `<span className={{'u-bold': name === lastname}}>{name}&nbsp;</span>` work?